.github/workflows/R-test-coverage.yml

```diff
@@ -0,0 +1,50 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
```

Comment:
Is this something that is a candidate for extraction into the `actions` repo?

Reply:
https://github.com/RMI-PACTA/actions/issues/6